extensions/sftp/tests/PutSFTPTests.cpp

```diff
@@ -506,7 +507,7 @@ TEST_CASE_METHOD(PutSFTPTestsFixture, "PutSFTP set mtime", "[PutSFTP]") {
 
   testFile("nifi_test/tstFile1.ext", "content 1");
   using namespace std::chrono;  // NOLINT(build/namespaces)
-  time_point<system_clock> modification_time = sys_days(January / 24 / 2065) + 5h + 20min;
+  system_clock::time_point modification_time = date::sys_days(date::January / 24 / 2065) + 5h + 20min;
```

Comment:
How is this change tied to this PR?

Reply:
I found this during development, because it didnt compile on centos (usually this is not a problem because -DSKIP_TESTS is hardcoded to be ON)

Reply:
Thanks for the info!